README.md

```diff
@@ -1,10 +1,10 @@
 This project provides declarative retry support for Spring
 applications. It is used in Spring Batch, Spring Integration,
 Spring for Apache Hadoop (amongst others).
+Imperative retry is also supported for explicit usage. 
 
 ## Quick Start
-
-Example:
+#### Declarative example:
```

Comment:
This is h3 heading I think? So three hashes?

Reply:
Initially thought about "something a bit greater than bold", but agree, this consistency is better:) Fixed.